aws/src/main/java/org/apache/iceberg/aws/lakeformation/LakeFormationAwsClientFactory.java

```diff
@@ -50,13 +45,17 @@
  * https://docs.aws.amazon.com/lake-formation/latest/dg/register-query-engine.html
  */
 public class LakeFormationAwsClientFactory extends AssumeRoleAwsClientFactory {
-
   public static final String LF_AUTHORIZED_CALLER = "LakeFormationAuthorizedCaller";
 
   private String dbName;
   private String tableName;
   private String glueCatalogId;
   private String glueAccountId;
+  private int maxCacheSize;
+  private long cacheExpirationInMillis;
```

Comment:
nit: `cacheExpirationInMillis` and `cacheExpiryLeadTimeInMillis` should match the config name, like `cacheRefreshLeadTimeMs`